Packs/Maltiverse/README.md

```diff
@@ -0,0 +1,248 @@
+## Overview
+---
+
+Analyze suspicious hashes, URLs, domains and IP addresses
+This integration was integrated and tested with version xx of Maltiverse
+## Maltiverse Playbook
+---
+
+## Use Cases
+---
+
```

Comment:
@altmannyarden Are there any use-cases we would like to put here?

Reply:
Yes, like I wrote in the design:
- Enriching information about different IOC types.
- Search for Reputation about different IOC types.
- Calculate DBot Score for indicators.